docs/_layouts/default.html

```diff
@@ -35,7 +35,7 @@
 			</a>
 			<a href="{{ site.baseurl }}/" alt="Hubble Enterprise">
 				<img src="{{ site.baseurl }}/assets/images/logo-title.svg" class="project-logo" width="3rem" height="3rem">
-				<h1 class="project-name"><script type="text/javascript">document.write(gheHostname())</script> Statistics</h1>
+				<h1 class="project-name">{{ site.url | remove: "http://", "https://" }} Statistics</h1>
```

Comment:
I get this error with Jekyll 3.3.1 locally:
```
  Liquid Exception: Liquid error (line 38): wrong number of arguments (given 3, expected 2) in /_layouts/default.html
```
Any idea what to do about it?

Reply:
Seems like "remove" only supports 1 argument, there doesn't seem to be a way on the Jekyll docs. We can try using both "remove" and "remove_first".